toolvoice/scripts/inference/cascade_asr/asr_cascade_openai_confetti.py

```
@@ -0,0 +1,453 @@
+            }))
+
+        # 3) Ask for a response
+        await ws.send(json.dumps({"type": "response.create"}))
+
+        # 4) Capture either tool-calls or text
+        async for raw in ws:
+            evt = json.loads(raw)
+            et = evt.get("type")
+
+            if et == "error":
+                raise RuntimeError(f"Realtime error: {evt}")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)